docs/releases/1.9.1.txt

```diff
@@ -11,3 +11,4 @@ Bugfixes
 
 * Fixed ``BaseCache.get_or_set()`` with the ``DummyCache`` backend
   (:ticket:`25840`).
+* Fixed an issue with nested ``ArrayField`` checks (:ticket:`25867`).
```

Comment:
I edited the PR title to say what I would write (describing the fix at a high level instead of the implementation details -- you could include those details on the second line if you want.) Similarly, here, I think we could be a bit more specific, "Fixed a system check crash with nested ArrayFields"